Dockerfile.build

```diff
@@ -27,6 +28,7 @@ ENV CMS_API=${CMS_API:-https://joplin.herokuapp.com/api/graphql/}
 ARG CMS_MEDIA
 ENV CMS_MEDIA=${CMS_MEDIA:-https://joplin.herokuapp.com/media/}
 
+COPY public /app/public
```

Comment:
saw this was in Dockerfile but not in Dockerfile.build -- just I added it. Not sure it's necessary at this moment tho.